README.md

```diff
@@ -53,8 +69,35 @@ Other interesting links are:
 * [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1037) **introductory course** in FIWARE Academy.
 * The [Contributing Guidelines](doc/contributing/contributing_guidelines.md) if your aim is to extend Cygnus.
 
-## Licensing
-Cygnus is licensed under Affero General Public License (GPL) version 3. You can find a [copy of this license in the repository](./LICENSE).
 
-## Reporting issues and contact information
+## Quality Assurance
```

Comment:
Same comment to the one published at https://github.com/telefonicaid/iotagent-ul/pull/301#discussion_r236292514